src/sdgio-edit.owl

```diff
@@ -2874,6 +2920,374 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020287> "2019-04-30T20:53:07Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020287> <https://clarisa.cgiar.org/api/srf-sub-idos/A.1.4>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020287> "enhanced capacity to deal with climactic risks and extremes (Mitigation and adaptation achieved)"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020237> <http://purl.unep.org/sdg/SDGIO_00020287>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020288> (enabled environment for climate resilience)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020288> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020288> "2019-04-30T20:53:26Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020288> <https://clarisa.cgiar.org/api/srf-sub-idos/A.1.5>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020288> "enabled environment for climate resilience"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020237> <http://purl.unep.org/sdg/SDGIO_00020288>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020289> (gender-equitable control of productive assets and resources)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020289> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020289> "2019-04-30T20:54:04Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020289> <https://clarisa.cgiar.org/api/srf-sub-idos/B.1.1>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020289> "gender-equitable control of productive assets and resources"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020237> <http://purl.unep.org/sdg/SDGIO_00020289>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020290> (technologies that reduce women`s labor and energy expenditure adopted)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020290> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020290> "2019-04-30T20:54:21Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020290> <https://clarisa.cgiar.org/api/srf-sub-idos/B.1.2>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020290> "technologies that reduce women`s labor and energy expenditure adopted"@en)
```

Comment:
> women`s 

the apostrophe here is kind of odd

Reply:
Thanks, @marieALaporte. I simply copied and pasted what the MARLO team put in the label column (after lower-casing everything that is not a proper noun). I let you and @pbuttigieg decide whether we should change the labels further. I also let @pbuttigieg to press the merge button when everything is ready.

Reply:
@Mttbnchtt @marieALaporte 
> women`s

Grammatically this is correct, but this is a backtick and not an apostrophe, which can cause all sorts of formatting chaos for parsers.

@htobon would there be an issue if we changed this to a true apostrophe? I assume MARLO would be working with the IRIs / codes rather than labels.

Reply:
> Shouldn't the content inside the parentheses be moved to a comment section and be removed from the label itself?

Yes, these should be moved out of the primary label as they're essentially restating and/or expanding on it. 

I would have suggested that you add each one as an [`alternative label`](http://purl.obolibrary.org/obo/IAO_0000118), but they are not semantically equivalent. Thus, I agree with @marieALaporte that you should add an rdfs:comment AP on the label stating: 

> The original wording of this IDO has a parenthetical extension to its label which reads: "[add phrase]". This has been removed for clarity and consistency."

Reply:
@Mttbnchtt The exceptions to this are when the parenthetical statement actually modifies something in the label. For example:
> land, water and forest degradation (Including deforestation) minimized and reversed

Should be left intact.

Reply:
Simply removing the parenthetical expansion would lead to different items having the same label. @pbuttigieg suggested the following solution:
1. Replace the expression "(more sustainably managed agro-ecosystems)" with "through more sustainably managed agro-ecosystems".
2. Replace the expression "(mitigation and adaptation achieved)" with "through the achievement of mitigation and adaptation plans".
I implemented such a solution modifying the following items:
- https://clarisa.cgiar.org/api/srf-sub-idos/3.3.2
- https://clarisa.cgiar.org/api/srf-sub-idos/3.3.2
- https://clarisa.cgiar.org/api/srf-sub-idos/A.1.1
- https://clarisa.cgiar.org/api/srf-sub-idos/A.1.4.

I also lower-cased the work "Including" in item https://clarisa.cgiar.org/api/srf-sub-idos/3.1.1.